src/Command/InfectionCommand.php

```diff
@@ -368,58 +371,65 @@ private function checkMetrics(): bool
 
     private function initContainer(InputInterface $input): void
     {
-        /** @var string|null $configFile */
-        $configFile = $input->hasOption('configuration')
-            ? trim((string) $input->getOption('configuration'))
-            : null
-        ;
+        // Currently the configuration is mandatory hence there is no way to
```

Comment:
We can probably move this comment to the doc block

Reply:
Why? It's only about the config file